en/SUMMARY.md

```diff
@@ -68,6 +68,9 @@
       * [Omnicopter](frames_multicopter/omnicopter.md)
       * [DJI F450 (CUAV v5+)](frames_multicopter/dji_f450_cuav_5plus.md)
       * [DJI F450 (CUAV v5 nano)](frames_multicopter/dji_f450_cuav_5nano.md)
+    * [Features](features_mc/README.md)
```

Comment:
FYI we're moving towards more vehicle-type centric docs. This is still a work in progress, but for that reason I have moved this, along with precision landing, into a section under multicopters.

For now this is just MC features, but probably will also have to link other interesting features that apply to all platforms.

Reply:
Nice, I think it's a good change! Before, I've struggled a bit to find where this feature would "fit" in the documentation, now it's clear.

Reply:
@zeroos Understandable.

Thanks for your review. I have integrated your comments into the introduction, so you can have another look if you like. Much appreciated.

Note, I expected to be wrong. But by writing all the assumptions a normal user might make, it forces you to correct me :-)

Reply:
Thank you, nice work, looks good to me!

I left just one minor comment about mentioning the distance to people, but overall I'm very happy with the result, thanks! :)

Reply:
Great. I've addressed that. @MaEtUgR says he is going to look at this very soon. With some luck today.